hazelcast/src/main/java/com/hazelcast/map/impl/operation/EntryOperation.java

```diff
@@ -85,10 +85,10 @@
  * If the EntryProcessor implements Offloadable the invocation scenario looks as follows:
  * - EntryOperation fetches the entry and locks the given key on partition-thread
  * - Then the processing is offloaded to the given executor
- * - When the processing finishes
+ * - Then the processing finishes
```

Comment:
Is this correct or was it a beginning of a new sentence just with improper padding?

Reply:
`When` seems more appropriate here, also having this as a html list item can be more clear.